doc/deps.md

```diff
@@ -217,17 +220,17 @@ mkdir <gtest_install_dir>/lib
 mv libgtest.a libgtest_main.a <gtest_install_dir>/lib/
 ```
 
-# Build CodeCompass
+# Step6 Build CodeCompass
 The dependencies which are installed manually because of known issues have to
 be seen by CMake build system:
 
 ```bash
-export GTEST_ROOT=<gtest_install_dir>
-export CMAKE_PREFIX_PATH=<thrift_install_dir>:$CMAKE_PREFIX_PATH
-export CMAKE_PREFIX_PATH=<clang_install_dir>:$CMAKE_PREFIX_PATH
-export CMAKE_PREFIX_PATH=<odb_install_dir>:$CMAKE_PREFIX_PATH
-export PATH=<thrift_install_dir>/bin:$PATH
-export PATH=<odb_install_dir>/bin:$PATH
+export GTEST_ROOT=<gtest_install_dir>                                      #Step5 <gtest_install_dir>
+export CMAKE_PREFIX_PATH=<thrift_install_dir>:$CMAKE_PREFIX_PATH           #Step1 <thrift_install_dir>
+export CMAKE_PREFIX_PATH=<clang_install_dir>:$CMAKE_PREFIX_PATH            #Step2 <clang_install_dir>
+export CMAKE_PREFIX_PATH=<odb_install_dir>:$CMAKE_PREFIX_PATH              #Step4 <odb_install_dir>
```

Comment:
If you put SQLite_ODB and Postgres_ODB and the likes into separate folders, this will not work, as at link time the libraries won't be bound properly.